code/basic.js

```diff
@@ -0,0 +1,485 @@
+  pictures: [
+    {
+      imageSrc: './assets/kittenpics/christina-yang-MJWxBOVjuUI-unsplash.jpg',
+      color: 'brown',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/daniel-park-N91GMm_6TBM-unsplash.jpg',
+      color: 'grey',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/dorothea-oldani-Hhm_fL04bE8-unsplash.jpg',
+      color: 'white',
+      environmentPreference: 'outdoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/fachy-marin-apO9wJ5caJc-unsplash.jpg',
+      color: 'white',
+      environmentPreference: 'outdoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/freddie-marriage-iYQC9xWMvw4-unsplash.jpg',
+      color: 'brown',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/joe-cleary-klH-f7mw2Ws-unsplash.jpg',
+      color: 'brown',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/ramiz-dedakovic-jerh2Nj1XWY-unsplash.jpg',
+      color: 'grey',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/raoul-droog-yMSecCHsIBc-unsplash.jpg',
+      color: 'grey',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/raoul-droog-yMSecCHsIBc-unsplash.jpg',
+      color: 'grey',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/sahand-babali-tjrN1-gF11o-unsplash.jpg',
+      color: 'black',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/timur-m-SAKLELG-pO8-unsplash.jpg',
+      color: 'white',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+    {
+      imageSrc: './assets/kittenpics/tran-mau-tri-tam-wiqbi_Uyvx8-unsplash.jpg',
+      color: 'white',
+      environmentPreference: 'indoors',
+      furState: 'furry',
+    },
+  ]
+}
+
+// Contains info about the created kitten
+const kitten = {
+  _name: '',
+  _gender: '',
+  _environmentPreference: '', 
+  _ownerName: '',
```

Comment:
It is not a wrong thing, but it is not a javascript convention to use underscore to create fields. You should use camelCase 😸